### PR TITLE
Revise upgrade guide for new workspace import UI

### PR DESCRIPTION
**Pull request by nfagerlund**
Not for merge until we ship the UI to saas.

### CODE REVIEW

File: content/source/docs/enterprise/upgrade/index.html.md [hunk 1]
```diff
@@ -10,85 +10,74 @@ If you used the legacy version of Terraform Enterprise (TFE), you probably have
 
 Follow these steps to migrate your old TFE environments to new TFE workspaces.
 
-## 1. Create a New Organization
+~> **Important:** Workspace migration imports the settings and the most recent Terraform state from a legacy environment. It does not import the state history or the run event history. If you need to preserve historical data from your legacy environments for regulatory reasons, please contact Hashicorp support for assistance and do not delete your legacy environments after migrating.
```
Comment (alisdair) [inline content/source/docs/enterprise/upgrade/index.html.md]: Hashicorp -> HashiCorp (and throughout)

File: content/source/docs/enterprise/upgrade/index.html.md [hunk 2]
```diff
@@ -10,85 +10,74 @@ If you used the legacy version of Terraform Enterprise (TFE), you probably have
...
-```terraform
-terraform {
-  backend "atlas" {
-    name = "<NEW_ORG>/<DESTINATION_WORKSPACE>"
-  }
-}
-```
+- In the "Legacy Environment" field, enter the environment to migrate, in the form `<LEGACY ORGANIZATION>/<ENVIRONMENT NAME>`.
+- In the "Workspace Name" field, enter the new name of the workspace, which should usually be the same as the old environment name.
+- Choose the same VCS provider and repository as the old environment, or choose "None" if you plan to [push configurations via the API](../workspaces/run-api.html).
```
Comment (alisdair) [inline content/source/docs/enterprise/upgrade/index.html.md]: Ah. [In the UI, we're linking to a different API page](https://github.com/hashicorp/atlas/blob/b97fb3cb3e0fe9d96c834034c5b2b0ab210bc5b5/frontend/atlas/app/templates/components/workspace-form.hbs#L108-L114), and this link is much more useful. Will fix the UI code!

File: content/source/docs/enterprise/upgrade/index.html.md (hunk 1 above)
Comment (nfagerlund) [reply content/source/docs/enterprise/upgrade/index.html.md]: OOPS D:

Comment (nfagerlund) [reply content/source/docs/enterprise/upgrade/index.html.md]: Rebased and fixed, wow that's an embarrassing one. I blame Bitbucket for giving me bad reflexes. 😅

File: content/source/docs/enterprise/upgrade/index.html.md [hunk 3]
```diff
@@ -97,35 +86,31 @@ The user account that owns this token must be a member of both the new and legac
...
 1. Edit this workspace's configuration so it reads state data from the new workspaces that replaced your legacy environments.
 2. At the same time, delete `ATLAS_TOKEN` from this workspace's variables.
 
 Make a note of any workspaces that access remote state as you migrate them, and update them soon after you finish migrating. Leaving a value for `ATLAS_TOKEN` over the long term can make your workspaces unnecessarily fragile.
 
-## 5. Disable VCS Integration in the Legacy Environment
+### Step 3.e: Unlock Workspace and Queue Plan
 
-Before committing and pushing the backend change to VCS, the legacy environment webhook must be disabled to prevent the configuration from running in both the legacy environment and the new workspace.
+Upon creation, the migrated workspace is locked to prevent runs. Unlock it after verifying the settings and variables, and queue a plan.
```
Comment (beekus) [inline content/source/docs/enterprise/upgrade/index.html.md]: As of the auto-locking changes, we also auto-unlock the workspace after the migration has finished.

Comment (beekus) [inline content/source/docs/enterprise/upgrade/index.html.md]: As of the auto-locking changes, we also auto-unlock the workspace after the migration has finished.

Comment (nfagerlund) [reply content/source/docs/enterprise/upgrade/index.html.md]: Oh! Glad to hear it, I'll update!

**Pull request merged**